nav2_bringup/nav2_gazebo_spawner/nav2_gazebo_spawner/nav2_gazebo_spawner.py

```diff
@@ -36,6 +36,8 @@ def main():
                         help='the y component of the initial position [meters]')
     parser.add_argument('-z', type=float, default=0,
                         help='the z component of the initial position [meters]')
+    parser.add_argument('-k', '--timeout', type=float, default=-1,
```

Comment:
What's -1 do? I think we want some timeout here - that's part of the reliability ticket to make sure that we're error tolerant

Reply:
Negative value for timeout blocks indefinitely until future is complete. What would be a decent timeout value for the spawn entity service? 1 second?

Reply:
Lets make it like 10s, enough to get the point across but still something that will fail cleanly.